about.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap" rel="stylesheet"> 
+    <link rel="stylesheet" type="text/css" href="style/about.css"/>
+    <title>About Firemen Meeting</title>
+</head>
+<body>
+
+  <header>
+    <ul class="logBar">
+      <il class="logBarItem"><img class="logBarImage" src="assets/about/facebook.svg"></il>
+      <il class="logBarItem"><img class="logBarImage" src="assets/about/twitter.svg"></il>
```

Comment:
- [ ] Please 🙏 add `alt` attribute for social icons images for accessibility purpose. 

> Alt tags provide context to what an image is displaying, informing search engine crawlers and allowing them to index an image correctly. However, not only do alt tags help search engine crawlers, not setting alt tags for your images can have a detrimental effect on the ranking of your web pages.